.github/workflows/ci.yml

```diff
@@ -85,3 +85,41 @@ jobs:
         with:
           severity: error
           ignore_paths: third-party
+  dokken-recipes:
```

Comment:
Please move this part in a separate PR.

Reply:
yes already moved: https://github.com/aws/aws-parallelcluster-cookbook/pull/1648